VERSION

```diff
@@ -1,3 +1,3 @@
 # The release version and build number of the stackdriver-agent package.
-PKG_VERSION=6.1.3
+PKG_VERSION=6.2.0
```

Comment:
Hmm, I have very contradicting feelings about this. I don't consider this to be a change in the API because the agent still behaves exactly the same for its purpose of collecting/sending metrics. The only change is to a config file that isn't fixed -- and that configurability is a "part of the API" (eh, sort of, maybe, not really, but you get it). That part of me wants to say that this doesn't even deserve a patch bump, but I do think there's merit in being able to differentiate between versions of the agent with the default changed vs not -- thus I'd vote patch bump.
(side question: if a user upgraded, their config wouldn't be overwritten with this right? (and yes yes, templates and new VMs that don't have one to begin with, inconsistencies etc etc))

If we /do/ consider a config file (that's shipped with the agent by default but configurable by the user by feature) to be a core part of the API, then I'd say this should be a major version bump.